docs/templates/sso-cd-https.adoc

```diff
@@ -224,3 +224,14 @@ for more information.
 
 
 
+
+[[tls]]
+== TLS/SSL configuration
+
+Red Hat Single Sign-On server can be configured to use TLS for handing incoming connections (also known as Key Store) and outgoing connections (also known as Trust Store). The configuration uses an automated script to convert a key or a certificate from PEM format into JKS, which is then consumed by Red Hat SSO.
```

Comment:
Typo, should be `handling` instead of `handing`.

Reply:
Fixed.